SimG4Core/Notification/interface/TrackInformation.h

```diff
@@ -82,71 +83,44 @@ class TrackInformation : public G4VUserTrackInformation {
 
   // methods for MTD info management
   //
-  inline void setFromTtoBTL() { mtdStatus_ |= 1 << 0; }  // 1st bit
-  inline bool isFromTtoBTL() const { return (mtdStatus_ >> 0) & 1; }
-  inline void setFromBTLtoT() { mtdStatus_ |= 1 << 1; }  // 2nd bit
-  inline bool isFromBTLtoT() const { return (mtdStatus_ >> 1) & 1; }
-  inline void setBTLdaughter() { mtdStatus_ |= 1 << 2; }  // 3rd bit
-  inline bool isBTLdaughter() const { return (mtdStatus_ >> 2) & 1; }
-  inline void setBTLlooper() { mtdStatus_ |= 1 << 3; }  // 4th bit
-  inline bool isBTLlooper() const { return (mtdStatus_ >> 3) & 1; }
+  void setFromTtoBTL() { mtdStatus_ |= 1 << 0; }  // 1st bit
```

Comment:
what is the specific reason to remove "inline"?

Reply:
Here the choice was to add "inline" everywhere up or remove from these new method. My choice was to keep "inline" if the implementation is in the bottom of this header , if implementation is together with the declaration I left it unchanged.

If you think that the choice was not optimal it is possible to put "inline" everywhere.

Reply:
I do not have a strong feeling, this can presumably be reviewed in a second round in case